podman/domain/images_manager.py

```diff
@@ -31,6 +32,8 @@
 
 logger = logging.getLogger("podman.images")
 
+builtin_list = list
+
```

Comment:
From my reading:

```
import builtins
>>> builtins.list
```

is more explicit about what is happening in this code. We are stuck with the naming collision. I think the safety of the typing is more important than the code bloat this will introduce. Or we can stay with List[].

Reply:
`typing.List` is deprecated, that's why we should move away from it if we can.

https://docs.python.org/3/library/typing.html#typing.List

I like your solution better though, it's more clear. I believe the `list` shadowing will not be problematic for users though, but the ambiguity in returns would eventually flag unexpected errors in IDEs, testing, checkers and such.

Reply:
@jwhonce I updated the PR with your suggestion

Reply:
@inknos The joys of trying to graph type checking on a language that is not statically typed.

Reply:
I know...